docs/dev-notes/2025-12-31/remove-abc-latest-20-rounds/plan.md

```
@@ -0,0 +1,106 @@
+# **ABCLatest20RoundsProvider の削除計画**
+
+## **目的**
+
+`ABCLatest20RoundsProvider` を削除し、関連するコードやテストを整理することで、コードベースの簡素化とメンテナンス性の向上を図る。
+
+## ** 理由**
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)